libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewOptions.java

```diff
@@ -23,7 +24,29 @@
 import com.mapbox.navigation.ui.voice.SpeechPlayer;
 
 @AutoValue
-public abstract class NavigationViewOptions extends NavigationUiOptions {
+public abstract class NavigationViewOptions {
+
```

Comment:
We 're missing some annotations here - not sure if the tool missed them 🤔

Reply:
I think it did because the class is abstract. I'd prefer to leave this as is though.